app/Resources/views/covers.html.twig

```diff
@@ -0,0 +1,21 @@
+{% set criticalCss = 'grid-listing' %}
+
+{% extends '::page.html.twig' %}
+
+{% block body %}
+
+    {% embed 'grid/listing-one-column.html.twig' with {fullWidth: true} %}
```

Comment:
Should probably just be `content`, definitely not a listing.

Reply:
@thewilkybarkid will change